src/Compilers/CSharp/Portable/Compilation/CSharpCompilation.cs

```diff
@@ -1523,6 +1523,9 @@ private MethodSymbol FindEntryPoint(CancellationToken cancellationToken, out Imm
             }
         }
 
+        internal override ImmutableArray<int> UnreferencedAssemblyIdentityDiagnosticCodes { get; }
+            = ImmutableArray.Create((int)ErrorCode.ERR_NoTypeDef);
+
```

Comment:
Is it worth storing this value in a static so we don't have an allocation per instance?